build/browser/app.js

```diff
@@ -273,7 +279,7 @@ driveScanner.service('DriveScannerService', function($q, DriveScannerRefreshServ
 
 });
 
-}).call(this,"/lib/browser/modules")
+}).call(this,"/lib\\browser\\modules")
```

Comment:
Uhm, I don't like how Browserify is compiling with backslashes on Windows, but not a problem I guess.